checkers/hugeParam_checker.go

```diff
@@ -57,14 +57,14 @@ func (*hugeParamChecker) isImplementStringer(decl *ast.FuncDecl) bool {
 		decl.Name.Name == "String" &&
 		decl.Type != nil &&
 		len(decl.Type.Params.List) == 0 &&
+		decl.Type.Results != nil &&
 		len(decl.Type.Results.List) == 1 &&
 		astcast.ToIdent(decl.Type.Results.List[0].Type).Name == "string" {
 		return true
 	}
 
 	return false
 }
-
```

Comment:
Please, leave the newline here. Thank you.

Reply:
Sorry about that, the removal of the line has been reverted.